src/python/pants/backend/pants_info/list_target_types_test.py

```diff
@@ -111,7 +111,7 @@ def test_list_all() -> None:
 
 
 def test_list_single() -> None:
-    class CustomField(BoolField):
+    class CustomField(TriBoolField):
```

Comment:
Oh tricky. The reason we want this to still be `TriBoolField` is that we want to test that `required = True` results in stripping `| None` from the help message. Without `required = True`, we would expect `| None` to be included.

Can you please add a comment to that effect?

Reply:
Yes!